components/kyma-environment-broker/common/orchestration/dto.go

```diff
@@ -153,3 +153,15 @@ type StatusResponseList struct {
 type UpgradeResponse struct {
 	OrchestrationID string `json:"orchestrationID"`
 }
+
+type MaintenancePolicyEntry struct {
+	GlobalAccountID string         `json:"globalAccountID"`
```

Comment:
Could you wrap GlobalAccountID and Plan into 
```
type MaintenancePolicyMatch struct {
	GlobalAccountID string
	Plan            string
}
```

and include it in the `MaintenancePolicyEntry` like this:
```
type MaintenancePolicyEntry struct {
	Match MaintenancePolicyMatch `json:"match"`
...
```

The reason is that we will enhance the policy later on with a NotMatch specification.